polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,260 @@
 ---
 title: Overview
-description: TODO
----
\ No newline at end of file
+description: Explore the fundamental concepts of the Polkadot protocol, including account addresses, extrinsics, events and runtime upgrades.
+---
+
+# Overview
+
+Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of tens of thousands. 
+
+Different projects have different value propositions, suggesting that the future will be multi-chain and that inter-chain communication will be crucial to establish collaborations and leveraging each other strengths.
```

Comment:
Just my own input.. not even multi-chain, but rather multi-_protocol_, but probably better to keep multi-chain for the sake of messaging ;)